File: booksAspCore/ClientApp/src/app/book.ts
import { BookType } from '../app/Models/bookType';
import { BookSubType } from '../app/Models/bookSubType';

export class Book {
    id: number;
    urlPic: string = 'http://books.ua/Product/GetImage?code_wares=33755';
    name: string;
    author: string;
    price: number;
    currency: string;
    countPage: number;
    language: string;
    description: string;
    bookType: BookType;
    bookSubType: BookSubType
}

File: booksAspCore/VSProdj/BookCoreApi/DAL/Entity/Book.cs
using DAL.Entity.Enums;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace DAL.Entity
{
    public class Book
    {
        public int Id { get; set; }
        public string UrlPic { get; set; } = "http://books.ua/Product/GetImage?code_wares=33755";
        public string Name { get; set; }
        public string Author { get; set; }
        [Column(TypeName = "decimal(18, 2)")]
        public decimal Price { get; set; }
        public string Currency { get; set; }
        public int CountPage { get; set; }
        public string Language { get; set; }
        public string Description { get; set; }
        public BookType BookType { get; set; }
        public BookSubType BookSubType { get; set; }
    }
}
